### PR TITLE
refactor: changed the naming to match the pending ERC spec

### PR DESCRIPTION
**Pull request by hexonaut**
A few people in the Ethereum community are putting together a spec for flash loans here: https://ethereum-magicians.org/t/flash-loan-eip-early-draft/4993. I've adjusted the language to match.

### CODE REVIEW

**Comment by hexonaut** [comment]
ERC3156 has been settled and merged: https://github.com/ethereum/EIPs/pull/3156

**Comment by brianmcmichael** [comment]
Nice!

File: src/flash.sol [hunk 1]
```diff
@@ -66,10 +66,10 @@ contract DssFlash {
         emit File(what, data);
     }
 
-    // --- Mint ---
-    function mint(
-        address _receiver,      // address of conformant IFlashMintReceiver
-        uint256 _amount,        // amount to flash mint [wad]
+    // --- flashLoan ---
```
Comment (lucas-manuel) [inline src/flash.sol:69]: Sorry - nit: `// --- Flashloan ---`

**Review by lucas-manuel — APPROVED** [review]
LGTM.

**Pull request merged**